editor/export/project_export.cpp

```diff
@@ -455,6 +478,75 @@ void ProjectExportDialog::_tab_changed(int) {
 	_update_feature_list();
 }
 
+void ProjectExportDialog::_patch_button_clicked(Object *p_item, int p_column, int p_id, int p_mouse_button_index) {
+	TreeItem *ti = Object::cast_to<TreeItem>(p_item);
+
+	patch_index = ti->get_metadata(0);
+
+	Ref<EditorExportPreset> current = get_current_preset();
+	ERR_FAIL_COND(current.is_null());
+
+	if (p_id == 0) {
+		Vector<String> preset_patches = current->get_patches();
+		ERR_FAIL_INDEX(patch_index, preset_patches.size());
+		patch_erase->set_text(vformat(TTR("Delete patch '%s' from list?"), preset_patches[patch_index].get_file()));
+		patch_erase->popup_centered();
+	} else {
+		patch_dialog->popup_centered_ratio();
```

Comment:
Default ratio is 0.8, which makes the dialog take most of the screen:
![image](https://github.com/user-attachments/assets/b6cb9e4f-c109-4e8a-bd54-9371db44de2c)
That's too big compared to typical file dialog.

Reply:
I changed it to `popup_file_dialog` instead, as well as in the add pack callback.